Packs/Twitter/Integrations/Twitterv2/Twitterv2.py

```diff
@@ -0,0 +1,394 @@
+                for item_in_include_media in include_media:
+                    media_key = item_in_include_media.get('media_key')
+                    for item_in_media_keys in media_keys:
+                        if item_in_media_keys == media_key:
+                            item_in_include_media.pop('public_metrics', None)
+                            item_in_data['media'] = item_in_include_media
+            for item_in_include_user in include_users:
+                id = item_in_include_user.get('id')
+                if id == author_id:
+                    item_in_data['author'] = item_in_include_user
+        next_token = response.get('meta', {}).get('next_token')
+        response.pop('includes', None)
+        response.pop('meta', None)
+    return data, next_token
+
+
+def create_context_data_get_user(response: dict) -> list[dict]:
+    """ Gets raw response form Twitter API and extracts the relevent data.
+        The data matched by pinned_tweet_id == includes.tweets.id
+            Args:
+                response: dict - raw response form Twitter API.
+            Returns:
+                dict: context data.
+    """
+    if response:
+        data: list[dict] = response.get('data', {})
+        include = response.get('includes', {})
+        for item_in_data in data:
+            entities = item_in_data.get('entities', {})
+            url = entities.get('url', {}).get('urls', {})
+            if entities and url:
+                for url_detail in url:
+                    url_detail.pop('start', None)
+                    url_detail.pop('end', None)
+                item_in_data['entities'] = url
+            pinned_tweet_id = item_in_data.get('pinned_tweet_id')
+            if pinned_tweet_id:
+                if include:
+                    include_tweets = include.get('tweets')
+                    for item_in_include_tweets in include_tweets:
+                        id = item_in_include_tweets.get('id')
+                        if pinned_tweet_id == id:
+                            item_in_data['Pinned_Tweets'] = item_in_include_tweets
+        response.pop('includes', None)
+        response.pop('meta', None)
+    return data
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    message: str = ''
+    try:
+        query_params = {'query': 'Twitter'}
+        client._http_request(method="GET", url_suffix='/tweets/search/recent',
+                             headers=client._headers, params=query_params, ok_codes=[200])
+        message = 'ok'
+    except DemistoException as e:
```

Comment:
Why do you need this? We saw that it never gets it, as we return an error from the api